scrapers/ManyVids.yml

```diff
@@ -98,7 +98,14 @@ jsonScrapers:
       Details: description
   sceneSearch:
     scene:
-      Title: vids.#.title
+      Title:
+       selector: vids.#.title
+       postProcess:
+        - replace:
+            - regex: '\["(\d+)",\s*"([^"]+)"\]'
+              with: $1/$2
+            - regex: ^
+              with: https://www.manyvids.com/Video/
```

Comment:
This seems to be misplaced unless we want URLs for titles in the search results 😅

Reply:
Lol, obviously something went wrong, updated the yml